e2e/utils.go

```diff
@@ -481,20 +481,57 @@ func enableTopologyInTemplate(data string) string {
 	return strings.ReplaceAll(data, "--feature-gates=Topology=false", "--feature-gates=Topology=true")
 }
 
-func validatePVCClone(sourcePvcPath, clonePvcPath, clonePvcAppPath string, f *framework.Framework) {
+// nolint:gocyclo // reduce complexity
+func validatePVCClone(sourcePvcPath, sourceAppPath, clonePvcPath, clonePvcAppPath string, f *framework.Framework) {
```

Comment:
The error that is currently reported:

    e2e/utils.go:485:1: cognitive complexity 42 of func `validatePVCClone` is high (> 40) (gocognit)

This is done by the `gocognit` linter. In addition to the `gocyclo` disabling, you would need to add a line like

    // nolint:gocognit // FIXME: this function is waaaaay to complex, split it in helper functions

Reply:
okay will try this or else will myself add helper function.